### ISSUE TITLE
Pascal voc segmentation challenge Submission of results

### ISSUE THREAD

**Issue by a56256324** [tensorflow/models#7940]
Hi, I am honored to use your code. When I downloaded your code and ran it, when submitting the results in pascal voc, I found that the result of val is consistent with the accuracy result you provided in pascal voc, but the test result is always different by 5-6%, so I want to ask you, is this normal? ? I used three machines with different configurations to verify, and found that it always differs from the results you submitted in the voc segmentation competition by 5-6%。

**Comment by a56256324** [comment]
@amahendrakar hi,Have you encountered the same problem? Maybe we can discuss it together before nobody answers this question.

**Comment by ravikyram** [comment]
@a56256324 

.Please, let us know which pretrained model you are using and share related code .Thanks!

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
